src/components/PublicGroupList.vue

```diff
@@ -154,6 +159,19 @@ export default {
             }
             return monitor.element.sendUrl && monitor.element.url && monitor.element.url !== "https://" && !this.editMode;
         },
+
+        /**
+         * Returns formatted certificate expiry or Bad cert message
+         * @param {Object} monitor Monitor to show expiry for
+         * @returns {string}
+         */
+        formatExpiry(monitor) {
+            if (monitor?.element?.validCert) {
+                return monitor.element.certExpiryDaysRemaining + " days";
+            } else {
+                return monitor.element.certExpiryDaysRemaining;
```

Comment:
If you rely on the server-generated message like this, `No/Bad Cert` will not be able to be translated. Is this an intentional decision?
(Could this be refactored to the frontend?)